tools/platformer/editor/export-tilemap-binary.lua

```diff
@@ -0,0 +1,79 @@
+-- Script to export tilemap data as a binary file.
```

Comment:
You should write up some brief instructions on how to use this, or at least note that lua is a dependency in `README.md`